cloudahoywriter.lua

```diff
@@ -334,6 +334,11 @@ local function dots_to_ones(dots)
     return dots / 2.5
 end
 
+-- Changes a positive to a negative and vice-versa.
+local function change_sign(number )
+    return -1 * number
+end
+
```

Comment:
FYI for Adi: this was a post-review change. CloudAhoy uses the opposite signs to X-Plane for deviation.